pushy-donkeys/Bangazon-Site
# Fixed Shopping Cart

**Changes Made:**
Added to Order Controller, fixed Shopping Cart button so that it takes you to Order Details depending on user Id.  Changed layout for Order.

**Steps to Test:**
```
git fetch --all
git checkout petesqueaky
```
Run program
Make sure that when you click Shopping Cart it goes to Order/Details
Make sure Shopping Cart shows product list of products that you have added
